applications/openshift/api-server/api_server_audit_log_path/rule.yml

```diff
@@ -45,9 +31,23 @@ ocil_clause: '<tt>audit-log-path</tt> does not contain a valid audit file path'
 
 ocil: |-
     Run the following command on the master node(s):
+    <pre>$ oc get configmap config -n openshift-apiserver -ojson | jq -r '.data["config.yaml"]' | jq '.apiServerArguments["audit-log-path"]'</pre>
+    The output should return a valid audit log path.
+
 {{%- if product == "ocp4" %}}
-    <pre>$ oc get configmap config -n openshift-kube-apiserver -ojson | jq -r '.data["config.yaml"]' | jq '.auditConfig["auditFilePath"]'</pre>
-{{% else %}}
-    <pre>$ sudo grep auditFilePath /etc/origin/master/master-config.yaml</pre>
+warnings:
+    - general: |-
+        {{{ openshift_cluster_setting("/api/v1/namespaces/openshift-apiserver/configmaps/config") | indent(8) }}}
 {{%- endif %}}
-    The output should return a valid audit log path.
+
+template:
+    name: yamlfile_value
+    vars:
+        ocp_data: "true"
+        filepath: '/api/v1/namespaces/openshift-apiserver/configmaps/config'
```

Comment:
I think this rule should check the kube apiserver.

Reply:
Got it. I will revert and improve how we are handling these rules.